### PR TITLE
Marcio/web analytics vitals explorer

### PR DESCRIPTION
**Pull request by marciocloudflare**

### CODE REVIEW

**Review by thepmer — APPROVED** [review]

File: products/analytics/src/content/web-analytics/understanding-web-analytics/core-web-vitals/index.md [hunk 1]
```diff
@@ -5,12 +5,64 @@ pcx-content-type: concept
...
+
+## Information collected
+
+Web Analytics uses its lightweight JavaScript beacon to collect the information Vitals Explorer uses. It does not use any client-side state, such as cookies or `localStorage`, to collect usage metrics. Vitals Explorer also does not “fingerprint” individuals via their IP address, User Agent string, or any other data.
+
+### Common data collected for all Core Web Vitals metrics
+
+* **Element**. A CSS selector representing the DOM node. With this string, you can use `document.querySelector(<element_name>)` in the dev console of your browser to find out which DOM node has a negative impact on your scores/values.
+* **Path**. The URL path at the time the Core Web Vitals are captured.
+* **Value**. The metric value for each Core Web Vitals. This value is in milliseconds for LCP or FID and a score for CLS (Cumulative Layout Shift).
```
Comment (cf-ypark) [inline products/analytics/src/content/web-analytics/understanding-web-analytics/core-web-vitals/index.md]: @marciocloudflare do we want to include how the score is calculated? maybe a link to this page? https://web.dev/cls/#layout-shift-score

**Review by cf-ypark — APPROVED** [review]
Fabulous! This looks really nice!!! I made one comment but it's up to you whether you think it's necessary to add that link (CLS).

File: products/analytics/src/content/web-analytics/understanding-web-analytics/core-web-vitals/index.md (hunk 1 above)
Comment (marciocloudflare) [reply products/analytics/src/content/web-analytics/understanding-web-analytics/core-web-vitals/index.md]: I can add it. Thanks!

File: products/analytics/src/content/web-analytics/understanding-web-analytics/core-web-vitals/index.md [hunk 2]
```diff
@@ -5,12 +5,64 @@ pcx-content-type: concept
...
+</Aside> 
 
-To improve your Core Web Vitals score, review each metric’s documentation: [Largest Contentful Paint](https://web.dev/optimize-lcp/), [First Input Delay](https://web.dev/optimize-fid/), [Cumulative Layout Shift](https://web.dev/optimize-cls/)
\ No newline at end of file
+## Web Analytics Vitals Explorer
+
+Vitals Explorer enables you to easily pinpoint which elements in a web page are affecting the user’s experience while browsing your website, in a visual form. 
+To find Vitals Explorer:
+
+1. Log in the [Cloudflare dashboard](https://dash.cloudflare.com/), and select your account. Go to **Web Analytics**.
```
Comment (abracchi-tw) [inline products/analytics/src/content/web-analytics/understanding-web-analytics/core-web-vitals/index.md]:
```suggestion
1. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com/) and select your account. Go to **Web Analytics**. 
```

File: products/analytics/src/content/web-analytics/understanding-web-analytics/core-web-vitals/index.md [hunk 3]
```diff
@@ -5,12 +5,64 @@ pcx-content-type: concept
...
 
-To improve your Core Web Vitals score, review each metric’s documentation: [Largest Contentful Paint](https://web.dev/optimize-lcp/), [First Input Delay](https://web.dev/optimize-fid/), [Cumulative Layout Shift](https://web.dev/optimize-cls/)
\ No newline at end of file
+## Web Analytics Vitals Explorer
+
+Vitals Explorer enables you to easily pinpoint which elements in a web page are affecting the user’s experience while browsing your website, in a visual form. 
+To find Vitals Explorer:
+
+1. Log in the [Cloudflare dashboard](https://dash.cloudflare.com/), and select your account. Go to **Web Analytics**. 
+1. Select your **website** > **Core Web Vitals**.
```
Comment (abracchi-tw) [inline products/analytics/src/content/web-analytics/understanding-web-analytics/core-web-vitals/index.md]:
```suggestion
1. Select your website and go to **Core Web Vitals**. 
```
`website` is not an actual UI element right?

File: products/analytics/src/content/web-analytics/understanding-web-analytics/page-load-time-summary/index.md [hunk 4]
```diff
@@ -13,4 +13,11 @@ pcx-content-type: reference
 * Processing (`domComplete` - `domInteractive`) - How long it took to render the page. This includes loading any resources that block page rendering, including images, scripts, and style sheets. If this number is big, optimize your document architecture, resource size, or configure settings in the Cloudflare Speed app, such as Auto Minify the source code. This document process can be drilled down more with `domInteractive`, `domContentLoadedEventStart`, `domContentLoadedEventEnd`, and `domComplete`.
 * Load Event (`loadEventEnd` - `loadEventStart`) - An event triggered by the browser when a document and its resources finish loading. The Load Event duration may be a useful metric if you have additional functions or any logic for the load event.
 
-![Web Analytics page load time](../../../static/images/dash-web_analytics-page_load_time.png)
\ No newline at end of file
+![Web Analytics page load time](../../../static/images/dash-web_analytics-page_load_time.png)
+
+## Data collected for Paint Timings
+
+Cloudflare need to collect some information to make Web Analytics work, and additional data is collected for Paint Timings:
```
Comment (abracchi-tw) [inline products/analytics/src/content/web-analytics/understanding-web-analytics/page-load-time-summary/index.md]: not entirely sure what you mean here

**Review by abracchi-tw — APPROVED** [review]

**Pull request merged**